au/code/au/CMakeLists.txt

```diff
@@ -0,0 +1,505 @@
+# Copyright 2024 Aurora Operations, Inc.
```

Comment:
To review this file, `"au/code/au/CMakeLists.txt"`, you can verify that it is identical to the contents of `"au/CMakeLists.txt"` on the parent branch.  GitHub UI does not make it obvious, but we simply moved this file to this new location.  (It doesn't show this because we had to retain a simpler version of the file at the old location.)